### PR TITLE
Initial Home Page Setup Merge to Master

### PR DESCRIPTION
**Pull request by MulesKC**
added HomeController, fragments.html, index.html, styles.css

### CODE REVIEW

File: src/main/resources/templates/index.html [hunk 1]
```diff
@@ -10,25 +10,41 @@
...
+        <!-- adds button for Nav Links when Navbar collapses (because screen size is small) -->
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="collapsibleNavbar">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+
+        <!-- Links on Navbar -->
+        <div class="collapse navbar-collapse" id="collapsibleNavbar">
+            <ul class="nav navbar-nav navbar-right">
+                <li class="nav-item">
+                    <a th:href="@{/register}" class="nav-link"><span class="glyphicon glyphicon-user"></span> Register</a>
```
Comment (KBeas21) [inline src/main/resources/templates/index.html:28]: STUPID LINK NAMENEEDING CHANGES

**Comment by jamiander** [comment]
Thumbs up!

**Review by scottpenneman — APPROVED** [review]

**Review by jamiander — APPROVED** [review]

**Review by hstrodtman — APPROVED** [review]

**Review by KBeas21 — APPROVED** [review]

**Review by KBeas21 — CHANGES_REQUESTED** [review]
remove merge conflicts that heather found

File: README.md [hunk 2]
```diff
@@ -41,8 +41,8 @@ provide a functioning website on which a phone app may be developed and improved
 - How to interact as a team using Github and Trello
 - How to interact with Google maps to display multiple pins and select a pin for directions
 
-#Setup
-- Creat user table in MySQL Workbench with one column labeled "id" as a primary key, not null, and auto-incremented.  The rest of the table will populate through persistence when a new user registers.
+# Setup
+- Create user table in MySQL Workbench with one column labeled "id" as a primary key, not null, and auto-incremented.  The rest of the table will populate through persistence when a new user registers.
```
Comment (KBeas21) [inline README.md:45]: Thanks for updating this Jamie!

File: src/main/resources/templates/fragments.html [hunk 3]
```diff
@@ -36,4 +36,15 @@
 </body>
 
 
-</html>
\ No newline at end of file
+</html>
```
Comment (KBeas21) [inline src/main/resources/templates/fragments.html]: This </html> needs to be removed as it was duplicated in the rebase (on line 50).

**Review by KBeas21 — APPROVED** [review]
Ryan and I synced up. These files look good now. We are going to be pushing this to master.

**Pull request merged**